Sources/ProjectDescription/ProjectOptions.swift

```diff
@@ -36,12 +36,38 @@ extension Project {
             disableSynthesizedResourceAccessors: Bool = false,
             textSettings: TextSettings = .textSettings(),
             xcodeProjectName: String? = nil
+        ) -> Self {
+            var bundleAccessorsOptions: BundleAccessorOptions = .enabled(includeObjcAccessor: true)
+            if disableBundleAccessors {
+                bundleAccessorsOptions = .disabled
+            }
+            return self.init(
+                automaticSchemesOptions: automaticSchemesOptions,
+                defaultKnownRegions: defaultKnownRegions,
+                developmentRegion: developmentRegion,
+                bundleAccessorsOptions: bundleAccessorsOptions,
+                disableShowEnvironmentVarsInScriptPhases: disableShowEnvironmentVarsInScriptPhases,
+                disableSynthesizedResourceAccessors: disableSynthesizedResourceAccessors,
+                textSettings: textSettings,
+                xcodeProjectName: xcodeProjectName
+            )
+        }
+
+        public static func projectOptions(
```

Comment:
I'm not sure if we want to deprecate the old initializers, I had to add an additional one to keep backward compatibility and allow compilation (default values in the functions caused a compilation issue)